### DISCUSSION TITLE
Feature Request: When session expires

### DISCUSSION THREAD

**Discussion by joeb1kenobe** [Security-Onion-Solutions/securityonion#3185] [Ideas] [answered]
When a session expires return to the logon page instead of showing the red banner at the top with error messages.

I know that it is easily fixed with a simple refresh of the page, and I am back at the logon screen. But it would be nice if the application would not show stale data with an error at the top of the page. 

Joe

**Comment by weslambert** [comment, marked as answer]
Created https://github.com/Security-Onion-Solutions/securityonion/issues/3222 for this.
